generated/nodejs/google-cloud-node/packages/monitoring/package.json

```diff
@@ -0,0 +1,21 @@
+{
+  "url": "https://github.com/googleapis/googleapis",
+  "name": "@google-cloud/monitoring",
+  "version": "0.7.1",
+  "author": "Google Inc.",
+  "description": "Monitoring library for Node.js",
```

Comment:
This needs to be hand-edited to 'Stackdriver Monitoring', please edit all the relevant places.